test2/test_ssl/test_nohttps.py

```diff
@@ -10,4 +10,5 @@ def test_http_is_forwarded(docker_compose, nginxproxy):
 def test_https_is_disabled(docker_compose, nginxproxy):
     with pytest.raises(ConnectionError) as excinfo:
         r = nginxproxy.get("https://web.nginx-proxy.tld/", allow_redirects=False)
-        assert "[Errno 93] Protocol not supportedd" in excinfo.value
+
+    assert "[Errno 93] Protocol not supported" in excinfo.value
```

Comment:
Yeah, I only noticed it because you had a typo in the assertion and it wasn't failing :D